library/java/net/openid/appauth/TokenRequest.java

```diff
@@ -185,6 +191,14 @@
     public final String scope;
 
     /**
+     * String value used to associate a Client session with an ID Token, and to mitigate replay attacks.
```

Comment:
Similarly, I'm also unclear whether the nonce is a defined parameter for the token request. I only see examples of it being sent to the authorization endpoint, not the token endpoint.